src/decorators/typeorm/SimpleTypeormLoader.ts

```diff
@@ -0,0 +1,181 @@
+import { TgdContext } from "#/types/TgdContext";
+import DataLoader from "dataloader";
+import { UseMiddleware } from "type-graphql";
+import Container from "typedi";
+import type { Connection } from "typeorm";
+import { RelationMetadata } from "typeorm/metadata/RelationMetadata";
+import { getToManyLoadData, getToOneLoadData, query } from "./utils";
+
+export function SimpleTypeormLoader<V>(): PropertyDecorator {
+  return (target: Object, propertyKey: string | symbol) => {
+    UseMiddleware(async ({ root, context }, next) => {
+      const tgdContext = context._tgdContext as TgdContext;
+      if (tgdContext.typeormGetConnection == null) {
+        throw Error("typeormGetConnection is not set");
+      }
+      const relation = tgdContext
+        .typeormGetConnection()
+        .getMetadata(target.constructor)
+        .findRelationWithPropertyPath(propertyKey.toString());
+
+      if (relation == null) {
+        return await next();
+      }
+
+      const dataloaderType = relation.isOneToOneOwner
+        ? OneToOneOwnerDataloader
+        : relation.isOneToOneNotOwner
+        ? OnetoOneNotOwnerDataloader
+        : relation.isManyToOne
+        ? ManyToOneDataloader
+        : relation.isOneToMany
+        ? OneToManyDataloader
+        : relation.isManyToMany
+        ? ManyToManyDataloader
+        : null;
```

Comment:
These Dataloaders might be able to integrate to fewer relations less than 5. But, I intended to divide each Dataloaders since it's useful to debug when loading data is failed in a specific relation.